source/pool/test/unit.js

```diff
@@ -456,6 +456,57 @@ describe('Pool Unit Tests', () => {
         )
       ).to.be.revertedWith('INVALID_TOKEN')
     })
+
+    it('withdrawWithSignature does not revert with signer who is admin', async () => {
```

Comment:
This can be named `withdrawWithSignature success` to align with similar withdraw function tests